functions/go/inflate-helm-chart/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/inflate-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins"
+	"sigs.k8s.io/kustomize/api/resmap"
+	"sigs.k8s.io/kustomize/api/resource"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
+	kyaml "sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+//nolint
+func main() {
+	asp := HelmChartProcessor{}
+	cmd := command.Build(&asp, command.StandaloneEnabled, false)
+
+	cmd.Short = "inflate helm chart"
```

Comment:
The `Short`, `Long` and `Examples` here should be the variables in functions/go/inflate-helm-chart/generated/docs.go

Reply:
fixed